pyproject.toml

```diff
@@ -71,49 +71,161 @@ line-length = 79
 indent-width = 4
 
 target-version = "py38"
+exclude = [
+  "CSV",
+  "ref.*?py",
+]
 
 [tool.ruff.lint]
-fixable = ["ALL"]
-preview = true
+fixable = []
+# preview = true
+# logger-objects = []
+task-tags = ["TODO", "FIXME", "XXX"]
 select = [
```

Comment:
Whatare your thoughts on including / not-including "C4" (flake8-comprehensions")

Reply:
Done!